src/components/ActionBar/README.md

```diff
@@ -1,5 +1,174 @@
 # ActionBar
```

Comment:
The biggest nuance of ActionBar is how it works between views on mobile. I think we should document putting the action bar layer at the root of the app and demonstrating the transitions between action bars.  Perhaps it's sufficient for now to link to the modal demo or lab?  But others should be able to construct the same code looking at this documentation / demo.

Reply:
added **keyed transitions** demo to https://square.github.io/maker/styleguide/desktop-modals/#/ActionBar which shows the transitions between a root actionbar and modal actionbar